index.html

```diff
@@ -13,13 +13,15 @@
     <header class="p-3 mb-3">
       <div class="container">
         <div
-          class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start"
+          class="row d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start"
```

Comment:
```suggestion
          class="row align-items-center justify-content-center justify-content-lg-start"
```

The `.row` class is `display: flex; flex-wrap: wrap` by default.